object_tracker/match.py

```diff
@@ -1,6 +1,8 @@
+import logging
 import cv2
 import numpy as np
 
+
 class Matcher:
     class Params:
```

Comment:
Add a 'TODO' note for us to fix Hydra, so we'll configure parameters better. 
In addition, for future work - try to figure out a better way to work with multiple parameterizations (i.e., PyTracking dimp18.py file with all of its params, and the parameters used by the OF and ORB.